src/model/__tests__/__snapshots__/Xref.spec.tsx.snap

```diff
@@ -16,29 +16,25 @@ exports[`XRef should render for ProteinProcessing section 1`] = `"<Fragment />"`
 
 exports[`XRef should render for Sequence section 1`] = `
 "<Fragment>
-  <XRefCategoryTable databaseCategory=\\"GENOME\\" databases={{...}} xRefData={{...}} accession=\\"P21802\\" />
-</Fragment>"
-`;
-
-exports[`XRef should render for Structure section 1`] = `"<Fragment />"`;
-
-exports[`XRefCategoryTable should render for database category GENOME and Ensembl 1`] = `
-"<div>
   <hr />
```

Comment:
The `<hr/>` also displays when there's nothing above (see O00311 Expression for instance). Would it be better to put it at the end of the `<Fragment>` block only if there's more `foundDatabaseCategory` to display?